src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -320,6 +328,150 @@ typedef struct {
+                              //!< (threshold) value of 16 for classifying a sequence of packet losses or discards as a burst.
+    UINT32 burstDiscardCount; //!< The cumulative number of bursts of discarded RTP packets, Appendix A (e) of [RFC8015].
+    DOUBLE burstLossRate; //!< The fraction of RTP packets lost during bursts to the total number of RTP packets expected in the bursts. As defined in
+                          //!< Appendix A (a) of [RFC7004], however, the actual value is reported without multiplying by 32768.
+    DOUBLE burstDiscardRate; //!< The fraction of RTP packets discarded during bursts to the total number of RTP packets expected in bursts. As
+                             //!< defined in Appendix A (e) of [RFC7004], however, the actual value is reported without multiplying by 32768.
+    DOUBLE gapLossRate;      //!< The fraction of RTP packets lost during the gap periods. Appendix A (b) of [RFC7004], however, the actual value is
+                             //!< reported without multiplying by 32768.
+    DOUBLE gapDiscardRate;   //!< The fraction of RTP packets discarded during the gap periods. Appendix A (f) of [RFC7004], however, the actual value
+                             //!< is reported without multiplying by 32768.
+    UINT32 framesDropped;    //!< Only valid for video. The total number of frames dropped prior to decode or dropped because the frame missed its
+                             //!< display deadline for this receiver's track. The measurement begins when the receiver is created and is a cumulative
+                             //!< metric as defined in Appendix A (g) of [RFC7004].
+    UINT32 partialFramesLost; //!< Only valid for video. The cumulative number of partial frames lost. The measurement begins when the receiver is
+                              //!< created and is a cumulative metric as defined in Appendix A (j) of [RFC7004]. This metric is incremented when the
+                              //!< frame is sent to the decoder. If the partial frame is received and recovered via retransmission or FEC before
+                              //!< decoding, the framesReceived counter is incremented.
+    UINT32 fullFramesLost; //!< Only valid for video. The cumulative number of full frames lost. The measurement begins when the receiver is created
+                           //!< and is a cumulative metric as defined in Appendix A (i) of [RFC7004].
+} RtcReceivedRtpStreamStats, *PRtcReceivedRtpStreamStats;
+
+/**
+ * @brief The RTCInboundRtpStreamStats dictionary represents the measurement metrics for the incoming RTP media stream. The timestamp reported in the
+ * statistics object is the time at which the data was sampled.
+ *
+ * Reference: https://www.w3.org/TR/webrtc-stats/#dom-rtcinboundrtpstreamstats
+ */
+typedef struct {
+    RtcReceivedRtpStreamStats received; // dictionary RTCInboundRtpStreamStats : RTCReceivedRtpStreamStats
```

Comment:
These do not seem to be part of inbound stats per spec. Can we add a comment to ensure we capture why we add RtcReceivedRtpStreamStats as part of inbound stats structure here?

Reply:
![image](https://user-images.githubusercontent.com/4367682/89668030-06417780-d892-11ea-91d1-aaab98338b24.png)